Dockerfile

```diff
@@ -24,5 +24,19 @@ RUN apt-get update && apt-get install -y --force-yes --no-install-recommends \
 ADD etc /etc
 ADD opt /opt
 
+# X11VNC_CLIP: The area of the application to show
+ENV X11VNC_CLIP "4096x4096+0+21"
+# The name of the application to publish with X11VNC
+ENV X11VNC_NAME "iKVM Viewer"
+# The delay (in secons) between connections before the container is terminated
+ENV CONSOLE_TTL "3600"
+
+# The address of the BMC to connect to
+ENV IPMI_ADDRESS "192.0.2.10"
```

Comment:
There is no default value for IPMI_ADDRESS anymore, this is commented in the Dockerfile to help understanding how to subclass this image.